src/App.tsx

```diff
@@ -21,11 +38,14 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={handleSelectChange}
+            defaultValue={5}
           >
-            <option value="3">3</option>
-            <option value="5">5</option>
-            <option value="10">10</option>
-            <option value="20">20</option>
+            {[3, 5, 10, 20].map(count => (
```

Comment:
better to move this array into some constant out of the component and reuse here